command/certificate/extractPE.go

```diff
@@ -0,0 +1,168 @@
+package certificate
+
+import (
+	"debug/pe"
+	"encoding/asn1"
+	"encoding/binary"
+	"encoding/pem"
+	"github.com/pkg/errors"
+	"github.com/smallstep/cli/errs"
+	"github.com/urfave/cli"
+	"go.mozilla.org/pkcs7"
+	"io"
+	"os"
+	"path"
+)
+
+var ErrCodeSignCertsNoCertificateTableFound = errors.New("No Certificate Table found")
+
+func extractPECommand() cli.Command {
+	return cli.Command{
+		Name:        "extract-pe",
+		Action:      cli.ActionFunc(extractPEAction),
+		Usage:       `TODO`,
+		UsageText:   `TODO`,
+		Description: `TODO`,
+	}
+}
+
+func extractPEAction(ctx *cli.Context) error {
+	if err := errs.MinMaxNumberOfArguments(ctx, 1, 1); err != nil {
```

Comment:
As only one argument is required we should use `errs.NumberOfArguments(ctx, 1)`

Reply:
I suppose for this use case doesn't make sense to allow an argument through stdin, right?

Reply:
Unfortunately no, the library needs an os.File and more specifically the io.ReadAt interface so it can go back and forth reading parts of the file.
I'm using it as part of another project, which is oriented toward streaming. The way I did it was to write all the file in a temporary location. Not so sure about storing it in a []byte because of the fact that it could be quite memory intensive.

But for this cli, yeah my guess is that stdin does not quite make sense